resources/configs/testnet/rollup_config.toml

```diff
@@ -16,7 +16,7 @@ network = "testnet"
 tx_backup_dir = ""
 
 [storage]
-path = "./db"
+path = "resources/dbs"
```

Comment:
some users will be downloading this toml and running binaries instead of cloning this repo, that's why I wanted a clearer directory which is just "db"

we can add db to gitignore though

Reply:
The same thing would apply to the devnet file in this case. However, for consistency, we should have it set this way. Since users will download this file, they can update the values as they please.

I have added a comment to clarify.